src/ontology/cl-edit.owl

```diff
@@ -4849,12 +4849,16 @@ SubClassOf(obo:CL_0000164 obo:CL_0000163)
 # Class: obo:CL_0000165 (neuroendocrine cell)
 
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "MESH:D055099"^^xsd:string) obo:IAO_0000115 obo:CL_0000165 "An endocrine cell that has the specialized function to produce and secrete hormones in response to neuronal signals."^^xsd:string)
+AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_0000165 "CL:0000381")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "BTO:0002691"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000165 "FMA:83810"^^xsd:string)
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000165 "neurosecretory cell"^^xsd:string)
+AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000165 "neurosecretory neuron"^^xsd:string)
 AnnotationAssertion(rdfs:label obo:CL_0000165 "neuroendocrine cell"^^xsd:string)
+EquivalentClasses(obo:CL_0000165 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0046879)))
```

Comment:
It looks like this was carried over from the obsoleted term, which came from the FBbt definition.

Consider adding comment from the reference in the ticket:

"The neurosecretory cell is neither an ordinary neuron nor an endocrine cell, but a combination of both. Its neuronal features resemble those of ordinary neurons concerning both structure and function. The production of a visible secretory material marks the neurosecretory neuron as a gland cell, and the fact that extractable cellular products act in the manner of hormones places it in the realm of endocrine elements."